.github/workflows/android-publish.yml

```diff
@@ -65,16 +80,17 @@ jobs:
       - name: Check for git tag
         if: ${{ inputs.BETA_RELEASE == 'Yes' }}
         run: |
-          cd android/scripts
+          cd ${{ inputs.ANDROID_SERVICE_LOCATION }}/scripts
```

Comment:
@adityankannan-tw 

Please remove /scripts as well.

As the ANDRIOD service location will contain the complete location